rules-reviewed/technology-usage/connect-technology-usage.windup.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0"?>
+<ruleset id="technology-usage-http" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides statistical summaries of the http items that were found during the analysis.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <phase>PostMigrationRulesPhase</phase>
+    </metadata>
+    <rules>
+        <rule id="technology-usage-connect-01000">
+            <when>
+                <graph-query discriminator="RMIServiceModel" />
+            </when>
+            <perform>
+                <technology-identified name="RMI">
+                    <tag name="Connect"/>
+                    <tag name="Other"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+
+        <rule id="technology-usage-connect-01100">
+            <when>
+                <or>
+                    <javaclass references="java.lang.System.{systemNativeMethods}({*})">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="java.lang.Runtime.load({*})">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="java.lang.Runtime.loadLibrary({*})">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="com.sun.jna.{*}">
```

Comment:
Good to identify also JNA usage that it's not mentioned in the mock-ups, but i think it should be better to have a rule dedicated to JNA (and so also a `technology-identified` tag for JNA) as JNA has been created to not use JNI ;)
`jni-native-code.windup.xml` rules take them together to address the compatibility of this native code with containers but from the technologies perspective they're two different things.

Reply:
Thanks for pointing that out